diff -up libshout-2.4.2/configure.ac.0000~ libshout-2.4.2/configure.ac
--- libshout-2.4.2/configure.ac.0000~	2019-04-29 15:36:24.000000000 +0200
+++ libshout-2.4.2/configure.ac	2019-05-02 02:09:23.527229971 +0200
@@ -9,7 +9,7 @@ m4_define(libshout_version, libshout_maj
 AC_INIT([libshout], libshout_version, [icecast-dev@xiph.org])
 AC_PREREQ([2.54])
 AC_CONFIG_SRCDIR([src/shout.c])
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS([config.h])
 # config.h guard
 AH_TOP([#ifndef __CONFIG_H__
 #define __CONFIG_H__ 1])
